browser.js

```diff
@@ -292,16 +274,15 @@ function zoomInit() {
 }
 
 document.addEventListener('DOMContentLoaded', () => {
-	// TODO: Figure out how to loud external scripts here
-	// [
-	// 	path.resolve('vendor/mousetrap.js'),
-	// 	path.resolve('vendor/mousetrap-global-bind.js')
-	// ].forEach(src => {
-	// 	const script = document.createElement('script');
-	// 	script.src = `file://${src}`;
-	// 	script.async = false;
-	// 	document.head.appendChild(script);
-	// });
```

Comment:
@danp Do you have any idea why the above didn't work? Maybe something to do with asar.

Maybe we could create our own custom protocol and load it through there. I think Atom does this. See: http://electron.atom.io/docs/api/protocol/